kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeCellMenu.js

```diff
@@ -357,7 +357,9 @@ define([
                         .append(
                             $('<div class="col-sm-8">')
                                 .append(
-                                    $('<div class="buttons pull-left">').append(this.$collapseBtn)
+                                    $(
+                                        '<div class="kb-cell-toolbar__buttons-container buttons pull-left">'
```

Comment:
update the class names -- not sure if this module is used anywhere, but if it is, the classname is still correct.